### ISSUE TITLE
 Is  the pre-trained model configured for the SPADE mode

### ISSUE THREAD

**Issue by qianyuling-wl** [zhanglonghao1992/One-Shot_Free-View_Neural_Talking_Head_Synthesis#78]
I apologize for the intrusion, but I wanted to inquire about the pre-trained model you provided. Is it configured for the SPADE mode? I'm encountering some issues while training in SPADE mode. The only change I made was to slightly decrease the 'num_repeats' parameter in the training settings. However, the model I trained is not producing the expected results; the mouth shapes don't seem to be driving correctly. Is there a difference between setting num_repeat to 75 with 200 epochs and setting num_repeat to 10 with 1500 epochs in the training configuration? I was wondering if you could possibly help me troubleshoot this issue. Attached are the results generated by the model along with the loss curve for your reference.
Thank you for your time and assistance.
![loss](https://github.com/zhanglonghao1992/One-Shot_Free-View_Neural_Talking_Head_Synthesis/assets/82922019/43e263be-2313-46b5-820b-481111164952)

https://github.com/zhanglonghao1992/One-Shot_Free-View_Neural_Talking_Head_Synthesis/assets/82922019/5e9a3515-223b-4212-815c-80fd6a758810

https://github.com/zhanglonghao1992/One-Shot_Free-View_Neural_Talking_Head_Synthesis/assets/82922019/9b5f3325-1859-4a52-bb48-2892bd0ab625

**Comment by renrenzsbbb** [comment]
> I apologize for the intrusion, but I wanted to inquire about the pre-trained model you provided. Is it configured for the SPADE mode? I'm encountering some issues while training in SPADE mode. The only change I made was to slightly decrease the 'num_repeats' parameter in the training settings. However, the model I trained is not producing the expected results; the mouth shapes don't seem to be driving correctly. Is there a difference between setting num_repeat to 75 with 200 epochs and setting num_repeat to 10 with 1500 epochs in the training configuration? I was wondering if you could possibly help me troubleshoot this issue. Attached are the results generated by the model along with the loss curve for your reference. Thank you for your time and assistance. ![loss](https://private-user-images.githubusercontent.com/82922019/282274829-43e263be-2313-46b5-820b-481111164952.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg0ODEzMjAsIm5iZiI6MTcyODQ4MTAyMCwicGF0aCI6Ii84MjkyMjAxOS8yODIyNzQ4MjktNDNlMjYzYmUtMjMxMy00NmI1LTgyMGItNDgxMTExMTY0OTUyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDA5VDEzMzcwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdjZTdhN2U3MjVhZjNkNjQwNmJkZGQ4NjI3MDBhNmY2MzdjZjJiZjE4YjZjMDI2NjgzOWRiNjBmMTY5MTdkZTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.x867AyMGdSX-oIPmJTN1SDN61BrdbsfEWDlg9p_c4R8)
> 
>  vox3.mp4 
>  vox5.mp4

have you finished above proble. If yes, can you share some experience?
